src/api/background/handlers/browser/tabs/tabs.handler.ts

```diff
@@ -0,0 +1,89 @@
+import Application from "~applications/application";
+import { getTab } from "~applications/tab";
+import {
+  getCachedAuthPopupWindowTabID,
+  resetKeepAlive,
+  resetPopupTabID
+} from "~utils/auth/auth.utils";
+import { createContextMenus } from "~utils/context_menus";
+import { getAppURL } from "~utils/format";
+import { updateIcon } from "~utils/icon";
+import { isomorphicSendMessage } from "~utils/messaging/messaging.utils";
+import browser from "webextension-polyfill";
+
+/**
+ * Handle tab updates (icon change, context menus, etc.)
+ *
+ * @param tabId ID of the tab to get.
+ */
+export async function handleTabUpdate(
+  tabID: number,
+  changeInfo?: browser.Tabs.OnUpdatedChangeInfoType
+) {
+  const popupTabID = getCachedAuthPopupWindowTabID();
+
+  if (popupTabID !== -1 && changeInfo?.status === "loading") {
+    isomorphicSendMessage({
```

Comment:
New event sent here to notify the Auth popup.

Reply:
👍